pkg/parse/events/builder.go

```diff
@@ -55,7 +57,10 @@ func (t *PublishingGroupBuilder) Build() []Publisher {
 		publishers = append(publishers, NewRetrySyncPublisher(t.Clock, t.RetryBackoff))
 	}
 	if t.StatusUpdatePeriod > 0 {
-		publishers = append(publishers, NewResetOnRunAttemptPublisher(StatusUpdateEventType, t.Clock, t.StatusUpdatePeriod))
+		// Status updates are on an independent timer from the sync event, and thus
+		// could run immediately after a sync event. The status updater performs a
+		// diff before making API calls so this should be low cost.
+		publishers = append(publishers, NewTimeDelayPublisher(StatusUpdateEventType, t.Clock, t.StatusUpdatePeriod))
```

Comment:
pkg/parse/events/doc.go needs updating to reflect this

Reply:
done